src/Backend/WASMWrapper.js

```diff
@@ -1,17 +1,19 @@
 import DataSeriesFactory from './DataSeriesFactory.js'
 import Module from './weblibdedx.js'
 
-
 /**
  * @typedef {LibdedxEntity}
  * @property {number} id - the libdedx ID assigned to the entity
  * @property {string} name - name of the entity read from libdedx
- */
-
-/**
- * @typedef {DataSeries}
- * @property {number[]} x - values to place on the X axis
- * @property {number[]} y - values to place on the Y axis
+ * 
+ * @typedef {PlotDataSeries}
+ * @property {number[]} energies - values of energy
+ * @property {number[]} powers - values of power
```

Comment:
this is a misleading name, too much in common with `raising to a power` or simply `power` in physics. I'd reccomend to switch to `xdata` and `ydata` or to `energies` + `stopping_powers`